src/scss/components/_button.scss

```
@@ -125,6 +145,8 @@ $button-padding-h: $grid-margin-2;
     $color-active: var(--of-button-anchor-active-fg),
     $color-background-active: var(--of-button-anchor-active-bg),
     $color-border-active: var(--of-button-anchor-active-color-border),
+    $color-border-focus: var(--of-button-anchor-focus-color-border),
```

there's no button-anchor variant for Utrecht, so we can fallback to the generic button design token here

```suggestion
    $color-border-focus: var(--of-button-anchor-focus-color-border, var(--utrecht-button-focus-border-color)),
```